src/test/java/org/fundacionjala/at15/pokemon/commands/SubCommandLearnTest.java

```diff
@@ -1,24 +1,32 @@
 package org.fundacionjala.at15.pokemon.commands;
 import static org.junit.Assert.assertEquals;
 
+import org.fundacionjala.at15.pokemon.Pokemon;
 import org.fundacionjala.at15.pokemon.commands.behavior.Learn;
+import org.fundacionjala.at15.pokemon.commands.behavior.Select;
 import org.fundacionjala.at15.pokemon.commands.create.CreatePokemon;
 import org.fundacionjala.at15.pokemon.io.Reader;
+import org.junit.Test;
+
 import picocli.CommandLine;
 
 public class SubCommandLearnTest {
-    //@Test
+    @Test
     public void commandShouldCreateAddMovementToJsonFile() {
-        CreatePokemon pokemon1 = new CreatePokemon();
-        new CommandLine(pokemon1).parseArgs("-n", "pikachu", "-hp", "200");
-        pokemon1.call();
+        CreatePokemon poke = new CreatePokemon();
+        new CommandLine(poke).parseArgs("-n", "pikachu", "-hp", "100");
+        poke.call();
+
```

Comment:
Before using a select command you must create a currents json files (crt-12345678):
`CreateCurrentEntities current = new CreateCurrentEntities();`
`new CommandLine(current);`
`current.call();`

Reply:
Thanks Adhemar! now is working the test